src/components/Profile/ProfileEdit/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+    Image, Grid, Container, Button, Form, Input, Segment, Divider,
+} from 'semantic-ui-react';
+import { NavLink } from 'react-router-dom';
+import './profile.scss';
+import PropTypes from 'prop-types';
+import imglove from '../Images/lov.jpeg';
+
+// const user = profile;
```

Comment:
Remove this line of commented code

Reply:
Hi @johnwayodi,
Thank you for this feedback.
I have taken note and made the adjustment.

Regards
Steven Ennis